script/lib/grd_string_replacements.py

```diff
@@ -0,0 +1,143 @@
+from lxml import etree
+import os
+import re
+
+
+# Strings we want to replace but that we also replace automatically for XTB files
+branding_replacements = [
+    (r'The Chromium Authors. All rights reserved.', r'The Brave Authors. All rights reserved.'),
+    (r'Google LLC. All rights reserved.', r'The Brave Authors. All rights reserved.'),
+    (r'The Chromium Authors', r'Brave Software Inc'),
+    (r'Google Chrome', r'Brave'),
+    (r'(Google)(?! Play)', r'Brave'),
+    (r'Chromium', r'Brave'),
+    (r'Chrome', r'Brave'),
+]
+
+
+# Strings we want to replace but that we need to use Transifex for
+# to translate the XTB files
+default_replacements = [
+    (r'Brave Web Store', r'Web Store'),
+    (r'Automatically send usage statistics and crash reports to Brave',
+     r'Automatically send crash reports to Brave'),
+    (r'Automatically sends usage statistics and crash reports to Brave',
+     r'Automatically sends crash reports to Brave'),
+    (r'You\'re incognito', r'This is a private window'),
+    (r'an incognito', r'a private'),
+    (r'an Incognito', r'a Private'),
+    (r'incognito', r'private'),
+    (r'Incognito', r'Private'),
+    (r'inco&gnito', r'&private'),
+    (r'Inco&gnito', r'&Private'),
+    (r'People', r'Profiles'),
+    # 'people' but only in the context of profiles, not humans.
+    (r'(?<!authenticate )people', r'profiles'),
+    (r'(Person)(?!\w)', r'Profile'),
+    (r'(person)(?!\w)', r'profile'),
+    (r'Bookmarks Bar\n', r'Bookmarks\n'),
+    (r'Bookmarks bar\n', r'Bookmarks\n'),
```

Comment:
Are these two lines in conflict with each other? Or maybe I'm not understanding how this works...

Reply:
just case sensitive replacement. Bar vs bar. Could be done with a regex probably too

Reply:
Ah, missed that!